src/components/transactions/Emode/EmodeModalContent.tsx

```diff
@@ -109,138 +129,273 @@ export const EmodeModalContent = () => {
+      <ArrowNarrowRightIcon />
+    </SvgIcon>
+  );
+
   if (txError && txError.blocking) {
     return <TxErrorView txError={txError} />;
   }
   if (emodeTxState.success) return <TxSuccessView action={<Trans>Emode</Trans>} />;
-
   return (
     <>
-      <TxModalTitle title="Efficiency mode (E-Mode)" />
+      <TxModalTitle title={`${mode} E-Mode`} />
       {isWrongNetwork && (
         <ChangeNetworkWarning networkName={networkConfig.name} chainId={currentChainId} />
       )}
 
-      {selectedEmode && selectedEmode.id !== 0 && (
-        <Alert severity="warning" sx={{ mb: 6 }}>
-          <Trans>
-            Enabling E-Mode only allows you to borrow assets belonging to the selected category
-            Stablecoins. Please visit our{' '}
-            <Link
-              href="https://docs.aave.com/faq/aave-v3-features#high-efficiency-mode-e-mode"
-              target="_blank"
-              rel="noopener"
-            >
-              FAQ guide
-            </Link>{' '}
-            to learn more about how it works and the applied restrictions.
-          </Trans>
-        </Alert>
+      {user.userEmodeCategoryId === 0 && (
+        <Warning severity="warning">
+          <Typography variant="caption">
+            <Trans>
+              Enabling E-Mode only allows you to borrow assets belonging to the selected category.
+              Please visit our{' '}
+              <Link
+                href="https://docs.aave.com/faq/aave-v3-features#high-efficiency-mode-e-mode"
+                target="_blank"
+                rel="noopener"
+              >
+                FAQ guide
+              </Link>{' '}
+              to learn more about how it works and the applied restrictions.
+            </Trans>
+          </Typography>
+        </Warning>
       )}
 
-      {Object.keys(emodeCategories).length > 2 && (
+      {showModal && (
         <EmodeSelect
           emodeCategories={emodeCategories}
-          selectedEmode={selectedEmode?.id || 0}
+          selectedEmode={selectedEmode?.id}
           setSelectedEmode={setSelectedEmode}
           baseAssetSymbol={currentNetworkConfig.baseAssetSymbol}
+          userEmode={user.userEmodeCategoryId}
         />
       )}
 
-      {blockingError !== undefined && <Alert severity="error">{handleBlocked()}</Alert>}
+      {blockingError === ErrorType.EMODE_DISABLED_LIQUIDATION && <Blocked />}
+      {blockingError === undefined && selectedEmode && selectedEmode.id === 0 && (
+        <Warning severity="error" sx={{ mt: 6, alignItems: 'center' }}>
+          <Typography variant="subheader1" color="#4F1919">
+            <Trans>Liquidation risk</Trans>
+          </Typography>
+          <Typography variant="caption">
+            <Trans>
+              This action will reduce your health factor. Please be mindful of the increased risk of
+              collateral liquidation.{' '}
+            </Trans>
+          </Typography>
+        </Warning>
+      )}
```

Comment:
Looks like the liquidation risk warning will always be shown, even if the users health factor isn't going to decrease. Will that cause confusion?

Reply:
Added logic to only display on HF decrease